### DISCUSSION TITLE
Long term budgets

### DISCUSSION THREAD

**Discussion by gccollect** [firefly-iii/firefly-iii#6952] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been discussed before.

### Description

From what I can see, budgets are limited to monthly at the most. However, it would be nice to be able to track long term projects such as home improvements which stretch over months if not years when including the planning applications. For example, you could set a fixed budget for the project and see how much is left.

### Solution

Allow non-resetting fixed budgets

### What are alternatives?

Use tags

**Comment by JC5** [comment]
That's already possible. Set a custom date range at the top of the page when visiting /budgets, the set the amount. It's not the easiest to do, but it works just fine.
